docs/FORD.OpenATF.SDD.TPL.md

```diff
@@ -0,0 +1,229 @@
+# INSTRUCTION
+------ Design document template explanation -------
+This is a SW Detailed Design template for each ATF component update.
+The original QCA template with more detail description is available at Luxoft portal
+https://adc.luxoft.com/confluence/display/PORTAL/Software+Detailed+Design+Template
+
+---------------------- HOWTO -----------------------
```

Comment:
@LuxoftAKutsan,
All this file is a big HOWTO - splitting header and text in the text will more mess up the user.